parachain/scripts/launch-network.sh

```diff
@@ -80,7 +80,7 @@ fi
 print_divider
 
 echo "launching zombienet network (in background), dir = $ZOMBIENET_DIR ..."
-$ZOMBIENET_BIN -d $ZOMBIENET_DIR -l silent spawn config.toml &
+nohup $ZOMBIENET_BIN -d $ZOMBIENET_DIR -l silent spawn config.toml > /dev/null 2>&1 &
```

Comment:
Hmm why do we need nohup? Isn't this process running in the background already?

Reply:
Without nohup, there is still dependency on that specific terminal, which logged in. 
It blocks the script to finish completely, even though it looks like everything inside script is already executed successfully.

Reply:
I see, so it's rather for jenkins instead of local setup?

Reply:
Yes. But it won't affect local setup. I have tested it.